### PR TITLE
Dotnet test

### PR DESCRIPTION
**Pull request by r1viollet**
# Description

Dotnet test.
The scaling by duration was not intuitive. I added as setting.

### CODE REVIEW

File: correctness_test.go [hunk 1]
```diff
@@ -143,8 +140,9 @@ func runTestApp(t *testing.T, dockerTag string, folder string) string {
 	if NETWORK_HOST {
 		args = append(args, "--network=host")
 	}
-	args = append(args, "-e", "DD_SERVICE=prof-correctness-"+folder)
+	args = append(args, "-e", "DD_SERVICE=prof-correctness-"+strings.Split(folder, "/")[1])
```
Comment (r1viollet) [inline correctness_test.go:143]: there was a bug where dotnet use the service name in the file name
the / was causing it to put things in a different directory

File: scenarios/dotnet_wall/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ENV LD_PRELOAD=/opt/datadog/continuousprofiler/Datadog.Linux.ApiWrapper.x64.so
+
+ENV DD_INTERNAL_PROFILING_OUTPUT_DIR=/app/data
+ENV DD_PROFILING_ENABLED=1
+ENV DD_TRACE_DEBUG=TRUE
+
+# Set the entry point for the container
+# ENTRYPOINT ["dotnet", "WorkloadScheduler.dll"]
+# ENTRYPOINT [ "bash" ]
+CMD dotnet WorkloadScheduler.dll && sleep 1
```
Comment (r1viollet) [inline scenarios/dotnet_wall/Dockerfile:54]: sleep is no longer needed

**Comment by r1viollet** [comment]
@realFlowControl I think this might break your remote PHP tests. I added a setting around the scaling by duration behaviour. Adding you as reviewer to check.

**Comment by realFlowControl** [comment]
Hey @r1viollet,
thanks for pinging me! This breaks the allocation and exception profiling tests, but I welcome this change as allocations and exceptions are way easier to have in absolute numbers compared to scaled by duration.
Let me setup a PR to add the `"scale_by_duration": true` to the PHP tests so that merging this PR will not make prof correctness fail in PHP

**Review by realFlowControl — APPROVED** [review]

**Pull request merged**